### DISCUSSION TITLE
Allow feeds to accept POST requests as feed content

### DISCUSSION THREAD

**Discussion by engram-design** [engram-design/FeedMe#893] [Ideas]
Will also include feed mapping and all other UI settings to be able to be set via the config file, as no 'feed' exists. Requires a reasonable refactor.

**Comment by clarknelson** [comment]
+1 sounds like a big change, even though it looks simple in Guzzle

**Comment by maxstrebel** [comment]
Is this something that can be done already? I need to query a Notion Database and am a bit stuck on how to do it.
